src/modules/Update.rb

```diff
@@ -850,6 +899,18 @@ def restore_backup
 
   private
 
+    # Includes the os-release file to backup
+    #
+    # @see https://www.freedesktop.org/software/systemd/man/os-release.html
+    def copy_os_release
+      ::FileUtils.cp(
+        Pathname.new("#{Installation.destdir}/etc/os-release"),
+        Pathname.new("#{Installation.destdir}/#{BACKUP_DIR}")
+      )
+    rescue
```

Comment:
Um, I'd avoid a catch-them-all rescue, this might also hide syntax errors in the code etc... Use some more specific exception here.